### PR TITLE
Add a footer with ENTRF & Netlify information

### LINKED ISSUE #117: Create bottom bar that contains ENTRF and Netlify icon
Create a bottom bar that will be displayed across all pages. The bottom bar will contain ENTRF Funding and Netlify icon.

### PR DESCRIPTION
**Pull request by UtkarshKr007**
Closes: #117

### CODE REVIEW

File: src/components/footer.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+								...GatsbyImageSharpFixed_noBase64
+							}
+						}
+					}
+				}
+			`}
+      render={data => (
+        <div className={classes.footer}>
+          <Container maxWidth="lg" className={classes.container}>
+            <Img title="ENRTF Logo" alt="ENRTF Logo" fixed={data.enrtfLogo.childImageSharp.fixed} />
```
Comment (floogulinc) [inline src/components/footer.js]: We should add the `fadeIn` prop set to false here to prevent the fade in.

File: src/components/footer.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+				query {
+					enrtfLogo: file(relativePath: { eq: "enrtf_logo.png" }) {
+						childImageSharp {
+							fixed(height: 50, quality: 70) {
+								...GatsbyImageSharpFixed_noBase64
+							}
+						}
+					}
+				}
+			`}
```
Comment (floogulinc) [inline src/components/footer.js]: Please use 2 space indentation and not tabs.

File: src/pages/index.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+html, body, #gatsby-focus-wrapper, #___gatsby {
+	height: 100%;
+	width: 100%;
+}
```
Comment (floogulinc) [inline src/pages/index.css]: Do we really need this extra CSS? Is styling the homepage container not enough? Also why does this style those gatsby IDs?

Comment (UtkarshKr007) [reply src/pages/index.css]: Yeah. Those are the IDs of two divs gatsby puts around every page, so even if we have a homepage container with styling of height: 100% , the container takes 100% of the height the wrappers allow it to which is not 100% of the page.

File: src/components/footer.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+export default function Footer() {
+  const classes = useStyles();
+  return (
+    <StaticQuery
+      query={graphql`
+				query {
+					enrtfLogo: file(relativePath: { eq: "enrtf_logo.png" }) {
+						childImageSharp {
+							fixed(height: 50, quality: 70) {
+								...GatsbyImageSharpFixed_noBase64
```
Comment (floogulinc) [inline src/components/footer.js]: We probably want to still use WebP

File: src/components/footer.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+				query {
+					enrtfLogo: file(relativePath: { eq: "enrtf_logo.png" }) {
+						childImageSharp {
+							fixed(height: 50, quality: 70) {
+								...GatsbyImageSharpFixed_withWebp_noBase64
+							}
+						}
+					}
+				}
+			`}
```
Comment (floogulinc) [inline src/components/footer.js]: Please don't use tab indents

**Review by floogulinc — CHANGES_REQUESTED** [review]

**Review by floogulinc — APPROVED** [review]

**Pull request merged**